content/blogposts/2022/cloud-managed-instances-for-enterprise.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Secure and scalable Sourcegraph Cloud managed instances for the enterprise"
+publishDate: 2022-09-27T10:00-07:00
+description: Sourcegraph Cloud, the single-tenant, dedicated Cloud version of Sourcegraph, is now generally available.
+tags: [blog]
+slug: "enterprise-cloud"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Soucegraph has long existed as self-hosted software, letting our customers confidently and securely run Sourcegraph in their own environments. After almost a decade of developing self-hosted Sourcegraph, we’ve gained the trust of some of the most sophisticated companies and development teams in the world, allowing us to work with their most sensitive IP: their code.
+
+At the same time, we’ve seen immense demand for Cloud-based Sourcegraph in recent years. Developers want a solution that can be spun up quickly, upgraded regularly, and maintained effortlessly without compromising security.
+
+In 2020, we experimented with deploying Cloud-based Sourcegraph to fit these needs, which we called “managed instances.” We ran these instances as single-tenant, isolated deployments in GCP, and the demand was so strong that almost 10% of our costumers are using this solution today. Since then, we’ve invested significantly in this single-tenant architecture solution, making it more scalable and bringing it to full feature parity with Sourcegraph Self-hosted. 
+
+Today—after years of building trust and engineering rigor—we are excited to move confidently to the Cloud with this highly secure and scalable solution. Dedicated Sourcegraph Cloud instances are now generally available, and the best way for new teams to use Sourcegraph
+
+## Security and scalability
+
+As a developer, your code is your most valuable IP. This makes security critical for Sourcegraph in all forms.
+
+That’s why we’re offering Sourcegraph Cloud via dedicated, single-tenant instances. When you run Sourcegraph Cloud—even as a trial—you get your own instance dedicated to only **your** code. You get company-level data isolation out-of-the-box.
+
+Plus, Sourcegraph has received a SOC 2 Type II report, so you can be confident that Sourcegraph controls for the security and confidentiality of your sensitive data.
+
+You also get enterprise-grade scaling. You can sync tens of thousands of repositories to your Sourcegraph instance and search across all of them without slowdowns, so you can stay in flow even when working in huge codebases.
```

Comment:
```suggestion
This means you get enterprise-grade scaling. You can sync tens of thousands of repositories to your Sourcegraph instance and search across all of them without slowdowns, so you can stay in flow even when working in huge codebases.
```

Reply:
We have a lot of "you also" through this post, so I'm updating where they run close together.